sky/cli.py

```diff
@@ -1706,13 +1717,40 @@ def status(all: bool, refresh: bool, show_spot_jobs: bool, clusters: List[str]):
                           show_all=False,
                           limit_num_jobs_to_show=not all,
                           is_called_by_user=False))
-        click.echo(f'{colorama.Fore.CYAN}{colorama.Style.BRIGHT}Clusters'
-                   f'{colorama.Style.RESET_ALL}')
+        if not ip:
+            click.echo(f'{colorama.Fore.CYAN}{colorama.Style.BRIGHT}Clusters'
+                       f'{colorama.Style.RESET_ALL}')
         query_clusters: Optional[List[str]] = None
         if clusters:
             query_clusters = _get_glob_clusters(clusters)
         cluster_records = core.status(cluster_names=query_clusters,
                                       refresh=refresh)
+        if ip:
+            if len(cluster_records) != 1:
+                with ux_utils.print_exception_no_traceback():
+                    raise RuntimeError(
+                        f'{len(cluster_records)} cluster found. Please specify '
+                        'exactly one cluster (or glob that only matches one '
+                        'cluster) to show its IP address.')
```

Comment:
Should we add a usage str here? 
```suggestion
                        f'{len(cluster_records)} cluster found. Please specify '
                        'exactly one cluster (or glob that only matches one '
                        'cluster) to show its IP address. \nUsage: `sky status --ip <cluster>`')
```